### ISSUE TITLE
Adjust CPP-generator include paths to C-generator ones

### ISSUE THREAD

**Issue by BeckmaR** [Yakindu/statecharts#867]
Consider the following partial folder structure inside a project:
* `src`
  * `sc_types.h`
* `src-gen`
  * `statechart.h`

In `statechart.h`, `sc_types.h` is included with
  `#include "sc_types.h"`, 
  where the C generator uses
  `#include "../src/sc_types.h"` 
  in the current SCT version.

 The latter type is less ambigious and should be used in the CPP generator as well. I did not attempt to compile a CPP project, so it is very possible that the output of the current CPP generator _works_ (maybe as long as there is only _one_ `sc_types.h` found in the includes and the project folder?), but the output of the C generator is better.

### tl;dr ###
In the C++ generator, change
  `#include "sc_types.h"`
  to
   `#include "../src/sc_types.h"`.

**Comment by tkutz** [comment]
Fixed with pull request #870
